### DISCUSSION TITLE
Can I install SQL server in sandboxie-plus?

### DISCUSSION THREAD

**Discussion by buptsad** [sandboxie-plus/Sandboxie#3849] [Q&A Feedback]
Installing SQL server on windows requires privilege to [Back up files and directories
](https://learn.microsoft.com/en-us/previous-versions/ms813696(v=msdn.10)?redirectedfrom=MSDN), [Manage auditing and security log](https://learn.microsoft.com/en-us/previous-versions/ms813959(v=msdn.10)?redirectedfrom=MSDN), [Debug programs](https://learn.microsoft.com/en-us/previous-versions/ms813847(v=msdn.10)?redirectedfrom=MSDN). Can I install it in sandboxie?

**Comment by bastik-1001** [comment]
There might be issues with the installer, if it can't be persuaded by `FakeAdminRights=y` to assume it would have the required permissions to make real changes. If it is going to install drivers, then it won't work, as drivers can't be installed into a sandbox.
